package.json

```diff
@@ -1342,7 +1342,7 @@
           },
           "default": [
             "-c",
-            "%DIR%/",
+            "%DIR%",
```

Comment:
On line 1349 it says:
`\n\nNote: At this moment -c option requires trailing slash.`
If the removal of the trailing slash has no consequences, the note should be removed.

Reply:
Your are right. The trailing `/` is still compulsory. See cmhughes/latexindent.pl/issues/180